ext/node/polyfills/internal/fs/handle.ts

```diff
@@ -152,7 +161,7 @@ function fsCall(fn, handle, ...args) {
     });
   }
 
-  return fn(handle, ...args);
+  return fn(handle.fd, ...args);
```

Comment:
Added a test for `fh.writeFile` to make sure this change is OK.

Reply:
I think we should rather let `fstat` accept `FileHandle`, which improves the compatibility further (See this change for reference https://github.com/denoland/deno/pull/25555 )

This `fsCall` util comes from this part in Node.js ( https://github.com/nodejs/node/blob/ccac4ee19d508abaf38bbabb87288ddeec7fcc21/lib/internal/fs/promises.js#L452-L470 ). I think it would be better to keep the code as aligned to node.js as possible.

Reply:
Nevermind. This looks just fine to me